jest-puppeteer.config.js

```diff
@@ -1,10 +1,23 @@
-module.exports = {
+/**
+ * @license BSD-3-Clause
+ *
+ * Copyright (c) 2019 Project Jupyter Contributors.
+ * Distributed under the terms of the 3-Clause BSD License.
+ */
+
+const config = {
   launch: {
-    headless: process.env.HEADLESS !== "false"
+    headless: process.env.HEADLESS !== 'false',
+    slowMo: process.env.SLOWMO === 'true'
```

Comment:
Nice

Reply:
Was definitely useful in seeing what button was being pressed in real-time.